### PR TITLE
[Frontend] Final fixes

### PR DESCRIPTION
**Pull request by carmen-lee**
[Frontend] Final fixes
> Team member: @carmen-lee 

## Expected Behavior
- Removed all warnings and log from app
- Restyled overlay for explore cards
- Removed navigation from notification items
- Changed login for app (jessica.jones@gmail.com) and made it case insensitive
- updated Explore cards content
- added alert box when registering a new account

simulator_screenshot_50743BF9-FE2F-446C-AE2B-4743A2EB0469

simulator_screenshot_42256313-0A0A-43E3-B610-83AE476A7989

simulator_screenshot_A20D4C22-12A4-4BB6-93EC-6B5E0B931293

### CODE REVIEW

**Review by cerasamson — APPROVED** [review]
The design changes and alert box look good. Thanks!

File: frontend/VolunteerOne/constants/loginCredentials.js
Comment (mattacy) [inline frontend/VolunteerOne/constants/loginCredentials.js:1]: this will look better for the demo

File: frontend/VolunteerOne/screens/Onboarding/Login.js
Comment (mattacy) [inline frontend/VolunteerOne/screens/Onboarding/Login.js:1]: thanks for this simple solution!

**Review by mattacy — APPROVED** [review]
Final touches look good for the demo

**Pull request merged**